### ISSUE TITLE
I can view dynamic data in the child name column on my dashboard table 

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#450]
## ✨ What new feature do you want to see?
The child name column on my dashboard displays dynamic data. 

## 🧶 Acceptance Criteria
- [x] I see the names of the children associated with my account
- [x] The above criteria work in both English and Spanish
- [ ] Verify accessibility (see details below)

## 🖍️ Implementation Details
This is currently blocked by #428. 

**Endpoint**: `GET /api/v1/case_list_for_dashboard`
**Authentication** required: yes
**Authorization**: scoped to the user's children
**Example response**:
```
[
    {
        "full_name": "Maria Baca",
        "approvals": [
            {
                "case_number": "5184465331"
            }
        ]
    },
    {
        "full_name": "K'Shawn Henderson",
        "approvals": [
            {
                "case_number": "5198908053"
            }
        ]
    }
]
```
> **NOTE TO ENGINEERS**
> 
> Please ensure that you are running Google Lighthouse and/or WebAIM (Wave) on any UI elements in your pull request to check for accessibility issues.
> 
> If it is out of scope to address those accessibility issues in your pull request (because the issue is in a parent component and needs refactoring to work, for example), **you are responsible for noting those issues in your pull request and writing bug tickets to address the issues - make sure you link the new tickets in your pull request**

## 🥧 Additional context
Figma link: https://www.figma.com/file/uCveVQQnYcw3MsenD1PNcG/P4P-2.0?node-id=2734%3A261

**Comment by rebeldroid12** [comment]
Hit the `api/v1/children` endpoint - returns of child objects - grab the key `full_name`
![Screen Shot 2020-10-25 at 7 11 53 PM](https://user-images.githubusercontent.com/5873894/97122711-f483a980-16f5-11eb-9115-e2e5e538979e.png)

**Comment by cjhaddad** [comment]
@csprayregen this will be QA-able when #557 is finished
